src/univariate/continuous/noncentralchisq.jl

```diff
@@ -1,4 +1,4 @@
-"""
+doc"""
```

Comment:
Otherwise one can escape the backslashes, I followed "doc_str should only be used when the docstring contains $ or \ characters that should not be parsed by Julia such as LaTeX syntax or Julia source code examples containing interpolation" from https://docs.julialang.org/en/latest/manual/documentation/